package.json

```diff
@@ -5,23 +5,28 @@
   "main": "index.js",
   "scripts": {
     "build": "webpack --production",
-    "start": "webpack-dev-server --content-base dist/"
+    "start": "webpack-dev-server --content-base dist/",
+    "dev": "webpack-dev-server --content-base dist/"
```

Comment:
duplicated command, if you want NPM to run start when dev types `npm run dev` you could change it to `"dev": "npm start"`